### PR TITLE
feat(manifest): support hard-coded last-release-sha

### PR DESCRIPTION
**Pull request by joeldodge79**
allow specifying an arbitrary sha to represent the last release. This is
helpful to use when trying to revert the last release PR. While you can
revert the changes from a bad release PR and delete associated tags/releases,
GitHub does not allow you to delete the actual PR so release-please will
still find it and consider it the last release.

worked for us in the wild: https://github.com/looker-open-source/sdk-codegen/pull/772

### CODE REVIEW

**Comment by joeldodge79** [comment]
just had another thought about this: I suppose a more specific solution to my problem would be supporting a PR label that disqualifies a PR from being considered as a release PR. thoughts?

File: src/manifest.ts [hunk 1]
```diff
@@ -337,11 +338,12 @@ export class Manifest {
     getFileMethod: 'getConfigJson' | 'getManifestJson',
     fileName: string
   ): Promise<{valid: true; obj: object} | {valid: false; obj: undefined}> {
-    let response: {valid: true; obj: object} | {valid: false; obj: undefined} =
-      {
-        valid: false,
-        obj: undefined,
-      };
+    let response:
+      | {valid: true; obj: object}
```
Comment (joeldodge79) [inline src/manifest.ts:342]: something weird going on here. This annotation claims it should be back in its original form. but in its original form gts complains and says that prettier wants it new-lined. ♾️

Comment (joeldodge79) [reply src/manifest.ts:342]: btw this is how `npm run fix` leaves it

Comment (sofisl) [reply src/manifest.ts:342]: You can try `rm -rf package-lock.json` and `rm -rf node_modules`, then `npm i` and `npm run fix`, see if that fixes your linting issue.

**Review by sofisl — APPROVED** [review]

File: src/manifest.ts (hunk 1 above)
Comment (joeldodge79) [reply src/manifest.ts]: oh interesting, after doing that `npm run fix` puts it back in the original form and `npm run lint` no longer complains about it. thanks for the tip!

**Review by bcoe — APPROVED** [review]

**Pull request merged**